src/Security/Authentication/Negotiate/src/PACKAGE.md

```diff
@@ -0,0 +1,49 @@
+## About
+
+`Microsoft.AspNetCore.Authentication.Negotiate` contains middleware to authenticate requests using Windows Authentication (also known as Negotiate, Kerberos, or NTLM).
```

Comment:
```suggestion
`Microsoft.AspNetCore.Authentication.Negotiate` contains an authentication handler used to authenticate requests using Windows Authentication (also known as Negotiate, Kerberos, or NTLM).
```

It's middleware in a general sense, but it's probably best to not imply it's middleware middleware.